dynacred/src/Contact.ts

```diff
@@ -746,12 +815,12 @@ class Recover {
     }
 
     findTrustee(trustee: InstanceID | Contact): number {
-        if (this.trusteesBuf == null || this.trusteesBuf.length === 0) {
+        if (this.trusteesBuf === undefined || this.trusteesBuf.length === 0) {
```

Comment:
`this.trusteesBuf === undefined` will always be false typewise; but as you're using the length of the buffer to mark missing value, maybe update `get trusteesBuf` to also return `undefined`.

Reply:
made `.*trustee.*` nicer and better.